chart/kubeapps/values.yaml

```diff
@@ -583,3 +580,7 @@ authProxy:
     requests:
       cpu: 25m
       memory: 32Mi
+
+# true for Helm 3; false for Helm 2:
+useHelm3: false # DOES NOT WORK YET; will act as a feature flag later on when we have Helm 3 support.
```

Comment:
I would move the `DOES NOT WORK YET` ( or something similar to the title instead of being an inline comment)

Reply:
Do you mean like this?

```diff
# true for Helm 3; false for Helm 2:
-useHelm3: false # DOES NOT WORK YET; will act as a feature flag later on when we have Helm 3 support.
+# DOES NOT WORK YET; will act as a feature flag later on when we have Helm 3 support.
+useHelm3: false
```

Reply:
Hm ... that feels pretty inline as well when I think about it.

Reply:
> Hm ... that feels pretty inline as well when I think about it.

I think he meant simply not having the comment on the same line after the text.

Anyway, updated and expanded the comment a bit).